frontend/components/viewpoints/pug/viewpoints.pug

```diff
@@ -19,14 +19,16 @@ div.viewpointsContainer
             ng-repeat="viewpoint in vm.viewpointsToShow track by $index")
 
             div.viewpointMain(layout="row")
-                img.viewpointThumbnail(ng-src="{{viewpoint.screenshot.thumbnailUrl}}")
+                img.viewpointThumbnail(ng-src="{{viewpoint.screenshot.thumbnailUrl}}" alt="")
                 div(flex, layout="column" layout-align="start")
                     div(
                         ng-if="vm.editSelectedView._id !== viewpoint._id").viewpointName {{viewpoint.name || "(Unnamed View)"}}
+                    label(style="visibility:hidden") Viewpoint Name
```

Comment:
Why do we need a hidden label here? Is the title not enough in textarea not enough?

Reply:
I chose to hide the label, so the screen reader can still see it but the user doesn't. 
As this would alter the existing design and look of the Viewpoints panel, i didn't want to these accessibility changes to change anything on the UI. 
The accessibility checker advised the following 
![labeltext](https://user-images.githubusercontent.com/6556019/46799317-704b6d00-cd4c-11e8-8665-d71b5c5d6fbf.png)
I opted for the latter as the textarea didnt have an ID so a title matching the label made more sense.